core/conn/odbc/src/odbc/nsksrvrcore/srvrothers.cpp

```diff
@@ -6591,6 +6612,10 @@ odbc_SQLSrvr_UpdateLob_sme_(
         exception_->u.ParamError.ParamDesc = SQLSVC_EXCEPTION_EXECUTE_FAILED;
     }
 
+    if (QryLobUpdateSrvrStmt != NULL) {
+        QryLobUpdateSrvrStmt->Close(SQL_CLOSE);
+    }
+
```

Comment:
This doesn't really close the internal cursor statements that SQL maintains to retrieve the LOB data. The JDBC driver has to issue a special syntax to close it. Once PR  https://github.com/apache/incubator-trafodion/pull/1216  is merged in the special syntax needs to be issued to close the stream output ( SQL extract )

Reply:
It looks like only one Extract statement can be active from a connection. Is that ok? What happens when there are two LOB columns in a table and that are being fetched simultaneously by the application